docs/catalyst_101.md

```diff
@@ -136,6 +136,153 @@ Once the existing members have voted to offer you membership, it’s up to you t
+
+### What we do now:
+
+We have three synchronous meetings:
+
+Sprint planning (Monday, week 1):
+
+- for each high level initiative, talk about each work item that we want to pull into this sprint
+  - what is it?
+  - should we do it?
+  - tag it with the sprint
+- any team discussions that people want to bring up
+
+Mid sprint checkin (Monday, week 2):
+
+- any team discussions that people want to bring up
+- for each item, make sure it's in the right place
+
+Sprint review (Thursday, week 2):
+
+- for each item tagged with the sprint, talk about:
+  - what it was
+  - what its status is
+- additional ad-hoc discussion topics
+
+Outside of meetings:
+
+- issues and PRs get ingested to the ZenHub board automatically
+- when issue status changes, move the issue in the ZenHub board (maybe - if not, we get to it in the meetings)
+
+### How we'll do it in GH Projects:
+
+Pretty much the same, but with the Catalyst Backlog project instead of ZenHub.
+
+Sprint planning (Monday, week 1):
+
+- for in-flight items in the old sprint:
+  - discuss status and whether we should roll into the new sprint; tag with new sprint if necessary.
+- for items in backlog:
+  - discuss whether we should pull into the new sprint; tag with new sprint if necessary.
+- ad-hoc discussion topics
+
+Mid sprint checkin (Monday, week 2):
+
+- for each item in the sprint:
+  - discuss status
+- ad-hoc discussion topics
+
+Sprint review (Thursday, week 2):
+
+- for each item in the sprint:
+  - discuss status
+- ad-hoc discussion topics
+
+Outside of sprint meetings:
+
+- Dazhong ingests new issues and PRs to the GH Projects board manually (don't worry, this takes like 5 minutes)
+- Everyone moves issues / linked PRs within the GH project when status changes
+
+### Current experiments
+
+For each experiment:
+
+- what's the goal?
+- what's the actual thing we do?
+- how will we assess success?
+- how do we know that sticking to it will feel better than giving up?
+
+#### Only pull scoped, prioritized tickets into the sprint
+
+Goal: make it easier to decide what to work on by agreeing on priority as a team and providing sufficient context for team members to get started on each item.
+
+Activity:
+
+Use the mid sprint checkin as a parallel work block to pull items into Backlog - aiming for at least 2N items in Backlog, where N is the number of people on the team. All items in Backlog should have scope and a next step:
+
+- scope: when would we consider this issue to be done? what criteria have to be met, or what behavior has to exist?
+- next step: what's the first step to getting this working?
+
+In the sprint review, instead of discussing status for each item in the sprint, prioritize the tickets that are in Backlog according to the following schema:
+
+- "Urgent" -> blocking work on high-priority tickets
```

Comment:
or affecting overall functioning of PUDL for users?

Reply:
I like having user-focused things here, but I'm a little wary of this specific language because it's very broad.

I'd like for Urgent to be a pretty clearly and tightly defined thing - otherwise lots of stuff goes into Urgent and then we have no prioritization scheme anymore 😢 

I think a few specific user-facing services that we provide being labeled "urgent" would be totally reasonable, though! Maybe worth cataloging exactly what sorts of stuff we do for people 😄